resources/template/home.html

```diff
@@ -0,0 +1,35 @@
+<html>
+<body>
+{{^userName}}
+<form action="/logout" method="post">
+    <button type="submit">Login</button>
+</form>
+
```

Comment:
Rename it templates not template.  That should allow it to run so you can meet the rest of the requirements.